README.md

```diff
@@ -1,6 +1,83 @@
-# visitAfrica-backend
-
-# Badges 
 
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed%20by-Hound-blueviolet)](https://houndci.com)
 [![Build Status](https://travis-ci.org/Soma-Technologies-Inc/visitAfrica-backend.svg?branch=develop)](https://travis-ci.org/Soma-Technologies-Inc/visitAfrica-backend)
+[![Coverage Status](https://coveralls.io/repos/github/andela/blackninjas-backend/badge.svg?branch=develop)](https://coveralls.io/github/Soma-Technologies-Inc/visitAfrica-backend?branch=develop)
+
+
+<a href="h" target="blank"><img align="center" src="https://raw.githubusercontent.com/tigthor/to-do-app/main/ezgif.com-gif-maker%20(2).gif" alt="tigthor" height="300" width="1200" /></a>
+
+
+# Visit Africa  :airplane: :earth_africa:
+Visit Africa Application is the application that will help users to visit Afirca by booking tickets to differents cities getting and paying for accomodations,rating their visits,chatting between users and work with their history of visits and Most importantly Visit Africa applicaion will promote african tourism.
+
+### Technologies :balance_scale: :computer:
+* [Es6+ Javascript](https://www.ecma-international.org/ecma-262/9.0/index.html) The project was written with ES6 syntax and above.
+* [Node/Express](https://nodejs.org/en/) A JavaScript runtime built on Chrome's V8 JavaScript engine.
+* [PostgreSQL](https://www.postgresql.org/) The World's Most Advanced Open Source Relational Database.
+* [Swagger](https://swagger.io/) Used for API documentation. Simplify API development for developers by helping to document APIs.
+* [Pivotal Tracker](https://www.pivotaltracker.com) A project management tool used to manage the app.
+* [Figma](https://www.figma.com/) Used for design mockups. It helps teams to design, prototype, and collaborate all in the browser.
```

Comment:
`Figma` has nothing to do with the backend, this should be included on the mobile repo

Reply:
Yeah I will  remove figma